Documentation/doc/resources/1.9.6/BaseDoxyfile.in

```diff
@@ -0,0 +1,812 @@
+#---------------------------------------------------------------------------
+
+# If the ALPHABETICAL_INDEX tag is set to YES, an alphabetical index of all
+# compounds will be generated. Enable this if the project contains a lot of
+# classes, structs, unions or interfaces.
+# The default value is: YES.
+
+ALPHABETICAL_INDEX     = NO
+
+#---------------------------------------------------------------------------
+# Configuration options related to the HTML output
+#---------------------------------------------------------------------------
+
+#
+# To get valid HTML the header file that includes any scripts and style sheets
+# that doxygen needs, which is dependent on the configuration options used (e.g.
+# the setting GENERATE_TREEVIEW). It is highly recommended to start with a
+# default header using
+# doxygen -w html new_header.html new_footer.html new_stylesheet.css
+# YourConfigFile
+# and then modify the file new_header.html. See also section "Doxygen usage"
+# for information on how to generate the default header that doxygen normally
+# uses.
+# Note: The header is subject to change so you typically have to regenerate the
+# default header when upgrading to a newer version of doxygen. For a description
+# of the possible markers and block names see the documentation.
+# This tag requires that the tag GENERATE_HTML is set to YES.
+
+HTML_HEADER            = ${CGAL_DOC_HEADER_PACKAGE}
+
+# The HTML_FOOTER tag can be used to specify a user-defined HTML footer for each
+# generated HTML page. If the tag is left blank doxygen will generate a standard
+# footer. See HTML_HEADER for more information on how to generate a default
+# footer and what special commands can be used inside the footer. See also
+# section "Doxygen usage" for information on how to generate the default footer
+# that doxygen normally uses.
+# This tag requires that the tag GENERATE_HTML is set to YES.
+
+HTML_FOOTER            = ${CGAL_DOC_RESOURCE_DIR}/footer.html
+
+# The HTML_EXTRA_STYLESHEET tag can be used to specify additional user-defined
+# cascading style sheets that are included after the standard style sheets
+# created by doxygen. Using this option one can overrule certain style aspects.
+# This is preferred over using HTML_STYLESHEET since it does not replace the
+# standard style sheet and is therefore more robust against future updates.
+# Doxygen will copy the style sheet files to the output directory.
+# Note: The order of the extra style sheet files is of importance (e.g. the last
+# style sheet in the list overrules the setting of the previous ones in the
+# list).
+# Note: Since the styling of scrollbars can currently not be overruled in
+# Webkit/Chromium, the styling will be left out of the default doxygen.css if
+# one or more extra stylesheets have been specified. So if scrollbar
+# customization is desired it has to be added explicitly. For an example see the
+# documentation.
+
+HTML_STYLESHEET        = ${CGAL_DOC_RESOURCE_DIR}/stylesheet.css
```

Comment:
Strange documentation for `HTML_EXTRA_STYLESHEET` and setting is called `HTML_STYLESHEET`

Reply:
That's from `doxygen -g`

Reply:
That is very strange, I have not been able to reproduce this with either `doxygen -g` nor with doxygen -u` 
(With the `doxygen -u` I get some warnings about an environment variable that replaces a complete settings line:
```
warning: ignoring unknown character '$' at line 610, file BaseDoxyfile.in
warning: ignoring unknown character '{' at line 610, file BaseDoxyfile.in
warning: ignoring unknown tag 'CGAL_OPT_EXTRACT_ALL_NO_DETAILED_IF_EMPTY' at line 610, file BaseDoxyfile.in
warning: ignoring unknown character '}' at line 610, file BaseDoxyfile.in
warning: ignoring unknown character '$' at line 1717, file BaseDoxyfile.in
warning: ignoring unknown character '{' at line 1717, file BaseDoxyfile.in
warning: ignoring unknown tag 'CGAL_DOC_MATHJAX_LOCATION_FULL_OPTION_LINE' at line 1717, file BaseDoxyfile.in
warning: ignoring unknown character '}' at line 1717, file BaseDoxyfile.in
```

Reply:
I just checked and it is there:
```
doxygen --version
1.9.6 (523d3509b8630cf9ee508f9196ff38555a8e18a4*)
```
The branch is based on
```
commit 4586b5cfaa3d46d51f6a51882951d15644c49edf (tag: Release_1_9_6)
Author: Dimitri van Heesch <doxygen@gmail.com>
Date:   Mon Dec 26 20:53:19 2022 +0100

    Update changelog for release 1.9.6
```

Reply:
-g

Reply:
But how did the things like `${CGAL_DOC_RESOURCE_DIR}/stylesheet.css` land in the file?
`doxygen -g` will just make a default `Doxyfile`.

Reply:
I misread you first comment, the doc is indeed not the right one,

Reply:
Looks better now, though `HTML_EXTRA_STYLESHEET` (although empty) is still missing, so it is not 100% based on a 1.9.6 Doxyfile.